src/pages/Index.vue

```diff
@@ -7,25 +7,31 @@
             <div class="rf-container">
                 <div class="rf-grid-row rf-grid-row--gutters">
                     <div class="rf-col">
-                        <h1>Start-ups, vous avez besoin de design !</h1>
-                        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Pariatur ipsa perferendis omnis, obcaecati possimus vero minus eius? Labore, eveniet, numquam.</p>
+                        <h1>Votre service numérique a besoin de design !</h1>
+                        <p>L'équipe de Designers Transverses est là pour vous aider à concevoir des services publics numériques intuitifs, accessible et simple d'usage.</p>
                     </div>
                 </div>
             </div>
             </section>
 
             <div class="rf-container">
+                <div class="rf-grid-row rf-grid-row--gutters">
+                    <div class="rf-col">
+                        <h1>Pour les Startups d'État</h1>
+                    </div>
```

Comment:
Je mettrais pas de titre ici, pour moi ça répond à la promesse faite juste au dessus (sans call to action)